core-feature-pack/common/src/main/resources/content/bin/installation-manager.sh

```diff
@@ -8,15 +8,25 @@ fi
 
 INSTALLATION_HOME="${1}"
 INST_MGR_LOG_PROPERTIES="${2}"
+INST_MGR_LOG_FILE="${3}"
 
 # For security, reset the environment variables first
 unset INST_MGR_COMMAND
 unset INST_MGR_STATUS
 
+TIMESTAMP=$(date "+%Y-%m-%d %H:%M:%S,%3N")
+LOG_NAME="[management-cli-installer]"
+
+log() {
+    echo "$TIMESTAMP ${1} $LOG_NAME - ${2}"
```

Comment:
This will log with the same timestamp for each entry. The `TIMESTAMP` variable should probably be removed and the `$(date "+%Y-%m-%d %H:%M:%S,%3N")` should be explicitly used here.

Reply:
Just a thought, but would `printf` work here too? Something like:

```bash
printf "%s %5s ${LOG_NAME} - %s\n" "$(date "+%Y-%m-%d %H:%M:%S,%3N")" "${1}" "${2}"
```

Reply:
@jamezp I guess it should work, my doubt comes from whether `printf` is available on all the shells, would it be? I tested it is available in dash in my local environment but I have not found how to validate it will be in all shells